Cauli/Cauli.swift

```diff
@@ -20,7 +20,7 @@
 //  THE SOFTWARE.
 //
 
-import Foundation
+import UIKit
```

Comment:
These are theoretically unrelated to this here, but I noticed it anyways and fixed it.